packages/api/tests/entities/Assignment.test.ts

```diff
@@ -0,0 +1,32 @@
+import { TestClient } from '../utils/TestClient';
+import { randomPath, randomModule, randomAssignment } from '../../src/Database/seeders/random';
+
+beforeAll(async () => { await TestClient.start(); });
+afterAll(async () => { await TestClient.stop(); });
+
+let moduleId: string;
+const setup = async () => {
+  await TestClient.resetDatabase();
+  await TestClient.workflowSignup();
+  const { id } = await TestClient.createPath(randomPath(undefined, 'path name'));
+  const { id: modId } = await TestClient.createModule(randomModule('name', id));
```

Comment:
We should add default random functions for each of the TestClient functions. That way we don't need to keep importing `randomXYZ` each time because it will default